packages/web-components/src/components/image-with-caption/image-with-caption.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import on from 'carbon-components/es/globals/js/misc/on';
+import '../image/image';
+import '../lightbox-media-viewer/lightbox-image-viewer';
+import '../button/button';
+import ZoomIn20 from 'carbon-web-components/es/icons/zoom--in/20';
+import 'carbon-web-components/es/components/modal/modal-close-button';
+import styles from './image-with-caption.scss';
+import ModalRenderMixin from '../../globals/mixins/modal-render';
+import Handle from '../../globals/internal/handle';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Image With Caption
+ *
+ * @element dds-image-with-caption
+ */
+
+@customElement(`${ddsPrefix}-image-with-caption`)
+class DDSImageWithCaption extends ModalRenderMixin(LitElement) {
+  /**
+   * The lightbox.
+   */
+  @property({ type: Boolean, reflect: true })
+  lightbox = false;
+
+  /**
+   * The image source.
+   */
+  @property({ reflect: true, attribute: 'default-src' })
+  defaultSrc = '';
+
+  /**
+   * The alt text.
+   */
+  @property({ reflect: true })
+  alt = '';
+
+  /**
+   * The heading.
+   */
+  @property({ reflect: true })
+  heading = '';
+
+  /**
+   * The description.
+   */
+  @property({ reflect: true })
+  copy = '';
+
+  /**
+   * `true` if the modal should be open.
+   */
+  @property({ type: Boolean, reflect: true })
+  open = false;
+
+  _handleClick() {
```

Comment:
Could you add a comment of what this method does ([ref](https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.11.0/packages/web-components/docs/coding-conventions.md#tsdoc-comments))?

```suggestion
  private _handleClick() {
```

Reply:
@asudoh updated!